gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/ReporterResource.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.management.handlers.management.api.resources.organizations;
+
+import io.gravitee.am.identityprovider.api.User;
+import io.gravitee.am.management.handlers.management.api.resources.AbstractResource;
+import io.gravitee.am.management.service.ReporterServiceProxy;
+import io.gravitee.am.model.Acl;
+import io.gravitee.am.model.Reference;
+import io.gravitee.am.model.ReferenceType;
+import io.gravitee.am.model.Reporter;
+import io.gravitee.am.model.permissions.Permission;
+import io.gravitee.am.service.OrganizationService;
+import io.gravitee.am.service.exception.OrganizationNotFoundException;
+import io.gravitee.am.service.exception.ReporterNotFoundException;
+import io.gravitee.am.service.model.UpdateReporter;
+import io.gravitee.common.http.MediaType;
+import io.reactivex.rxjava3.core.Completable;
+import io.reactivex.rxjava3.core.Maybe;
+import io.reactivex.rxjava3.core.Single;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.inject.Inject;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.ws.rs.BadRequestException;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.PUT;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.container.AsyncResponse;
+import jakarta.ws.rs.container.Suspended;
+import jakarta.ws.rs.core.Response;
+
+import java.util.Optional;
+
+/**
+ * @author Titouan COMPIEGNE (titouan.compiegne at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class ReporterResource extends AbstractResource {
+
+    private final ReporterServiceProxy reporterService;
+
+    private final OrganizationService organizationService;
+
+    @Inject
+    public ReporterResource(ReporterServiceProxy reporterService, OrganizationService domainService) {
+        this.reporterService = reporterService;
+        this.organizationService = domainService;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get a reporter",
```

Comment:
**TODO:** can you please add an operationId for all the method to avoid collision with the reporters operation on domain level during code generation based on the OpenAPI Specification ?